src/core_plugins/kibana/public/dashboard/index.js

```diff
@@ -26,6 +26,5 @@ uiRoutes
   })
   .when('/dashboard/:id', {
```

Comment:
I kind of wish that this was the "view" url, and then `/dashboard/:id/edit` was the url for "edit" mode...